master/buildbot/test/unit/test_steps_shell.py

```diff
@@ -421,7 +430,11 @@ def test_run_property_no_strip(self):
         self.expectOutcome(result=SUCCESS,
                            state_string="property 'res' set")
         self.expectProperty("res", "\n\nabcdef\n")
-        self.expectLogfile('property changes', r"res: u'\n\nabcdef\n'")
+        if PY3:
```

Comment:
I think it would be wiser to have same behaviour on py2 and py3, and probably the one that makes more sense is

    res = r"res: '\n\nabcdef\n'"

I am not sure exactly what are the implication though. can you look at that?